File: PythonCode/1070.py
#Questão: Seis Números Ímpares
x = int(input())
if (x % 2 == 0):
    x += 1
    for i in range(x,x + 12,2):
        print(i)
else:
    for i in range(x,x + 12, 2):
        print(i)

File: C++Code/1070.cpp
//Questão: Seis Números Ímpares
#include <iostream>

using namespace std;

int main()
{
	int i, n;
	cin >> n;
	
	
    if (n%2 == 0)
    {
        n += 1;
        for(i = n; i <= n+10; i += 2){
              cout << i << endl;
        }
      
    }else 
    {
        cout<< "hdds"<< endl;
        for(i = n; i <= n+10; i += 2){
              cout << i << endl;
        }
    }
    return 0;
	
}